### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by babayoshihiko**
- R version 4.3.2 and the corresponding announcement
- 2023 rather than 2022?
- **C**reative

Twitter may be X? I did not fix it, though.

### CODE REVIEW

File: 02-spatial-data.Rmd [hunk 1]
```diff
@@ -8,7 +8,7 @@
 This is the first practical chapter of the book, and therefore it comes with some software requirements.
 <!-- toDo: rl-->
 <!-- should we be that specific regarding the r version?-->
-You need access to a computer with a recent version of R installed (R [4.3.0](https://stat.ethz.ch/pipermail/r-announce/2022/000683.html) or a later version).
+You need access to a computer with a recent version of R installed (R [4.3.2](https://stat.ethz.ch/pipermail/r-announce/2023/000697.html) or a later version).
```
Comment (Robinlovelace) [inline 02-spatial-data.Rmd:11]: That version is a lot later, why would you recommend this version? I see no problem with asking for up-to-date R versions though.

File: 02-spatial-data.Rmd [hunk 2]
```diff
@@ -158,7 +158,7 @@ Simple Features is a hierarchical data model that represents a wide range of geo
 Of 18 geometry types supported by the specification, only 7 are used in the vast majority of geographic research (see Figure \@ref(fig:sf-ogc));
 these core geometry types are fully supported by the R package **sf** [@pebesma_simple_2018].^[
 The full OGC standard includes rather exotic geometry types including 'surface' and 'curve' geometry types, which currently have limited application in real world applications.
-All 18 types can be represented with the **sf** package, although at the time of writing (2022) plotting only works for the 'core 7'.
+All 18 types can be represented with the **sf** package, although at the time of writing (2023) plotting only works for the 'core 7'.
```
Comment (Robinlovelace) [inline 02-spatial-data.Rmd:161]: :+1: 2024 soon!

File: index.Rmd [hunk 3]
```diff
@@ -73,7 +73,7 @@ if (is_on_ghactions){
 ```{asis index-2-3, echo=is_html}
 <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-nd/4.0/88x31.png"/></a><br/>This book is licensed to you under a <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License</a>.
 
-<a rel="license" href="https://creativecommons.org/publicdomain/zero/1.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/zero/1.0/88x31.png"/></a><br/>The code samples in this book are licensed under <a rel="license" href="https://creativecommons.org/publicdomain/zero/1.0/">reative Commons CC0 1.0 Universal (CC0 1.0)</a>.
+<a rel="license" href="https://creativecommons.org/publicdomain/zero/1.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/zero/1.0/88x31.png"/></a><br/>The code samples in this book are licensed under <a rel="license" href="https://creativecommons.org/publicdomain/zero/1.0/">Creative Commons CC0 1.0 Universal (CC0 1.0)</a>.
```
Comment (Robinlovelace) [inline index.Rmd:76]: :+1:

**Review by Robinlovelace — APPROVED** [review]
Looks good to me. Thoughts @Nowosad and @jannes-m ?

**Comment by Nowosad** [comment]
:+1 from me

**Pull request merged**